go/src/koding/kites/kloud/utils/object/object.go

```diff
@@ -69,6 +72,66 @@ func (b *Builder) Build(v interface{}, ignored ...string) Object {
 	return obj
 }
 
+// TODO(rjeczalik): add support for patching - accept path instead of key,
```

Comment:
If you mean https://github.com/koding/koding/pull/9182#discussion_r82774059 then it does not work for different types.